value_sensors/template.lua

```diff
@@ -9,8 +9,13 @@ function ValueSensor.new(name)
         ["name"] = name,
         ["display_name"] = { "sensor."..name..".name" },
         ["format_key"] = "sensor."..name..".format",
+        ["color"] = nil,
```

Comment:
Initializing a table key with a nil value is a no-op in Lua, to the best of my current knowledge, though I may have written this same kind of initializer myself. May I instead recommend initializing with a good default (like `["color"] = { r=1, g=1, b=1, a=1 }`) and just assuming in `sensor:update_ui` that it will be present?